### PR TITLE
Feature/ui

### PR DESCRIPTION
**Pull request by walimike**
**This PR adds UI content for fast-food-fast app**
The UI pages can be viewed directly here; https://walimike.github.io/fast-food-fast/ 
 @joelethan, @KicaRonaldOkello, @jeanjoe, @malmike, @billkabanga please give me some feedback, thank you.

### CODE REVIEW

**Comment by Luleherll** [comment]
i think you should remove the transparency on` user and admin`. the items can't be seen clearly

File: UI/adminhome.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+          <table>
+            <tr><th></th></tr>
+          </table>
+          <button type="submit" class="button" onClick="clearDiv(); markOrder(); refreshPage();" >Complete</button>
+        </div>
+      </div>
+      <div id="admincart2">
+        <h1>List of Food Items</h1>
+        <div id="foodlist">
+          <table>
```
Comment (jeanjoe) [inline UI/adminhome.html]: Please use `<tr>` and `<td>` inside `<table>` and then you can add other HTML tags like`<p>`.

**Review by jeanjoe — CHANGES_REQUESTED** [review]

File: UI/adminhome.html (hunk 1 above)
Comment (walimike) [reply UI/adminhome.html]: Oh okay, thank you

File: UI/signup.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <meta name="viewport" content="width=device-width">
+    <meta name="author" content="Michael Robert Wali">
+    <title>Admin</title>
+    <link href="css/style.css" type="text/css" rel="stylesheet">
+    <script type="text/javascript" src="js/main.js"></script>
+  </head>
+  <body>
+    <section class="layout">
+      <div class="branding">
+        <h1>Fast Food <span class="highlight">Fast</span><h1>
```
Comment (jeanjoe) [inline UI/signup.html]: You did not close `<h1>` tag

File: UI/index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    <meta name="author" content="Michael Robert Wali">
+    <title>FastFoodFast</title>
+    <link href="css/style.css" type="text/css" rel="stylesheet">
+    <script type="text/javascript" src="js/main.js"></script>
+  </head>
+  <body>
+    <!-- section layout for index.html -->
+    <section class="layout">
+      <div class="branding">
+        <h1>Fast Food <span class="highlight">Fast</span><h1>
```
Comment (jeanjoe) [inline UI/index.html]: Close `<h1>` tag

File: UI/userhome.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <meta name="author" content="Michael Robert Wali">
+    <title>User</title>
+    <link href="css/style.css" type="text/css" rel="stylesheet">
+    <script type="text/javascript" src="js/main.js"></script>
+  </head>
+  <div class="container">
+  <body>
```
Comment (jeanjoe) [inline UI/userhome.html]: You should not have `<body>` tag inside a `<div>`.  Body should always come first after `</head>`

**Review by jeanjoe — CHANGES_REQUESTED** [review]

**Comment by walimike** [comment]
Merges UI commits to develop

**Pull request merged**